readme.md

```diff
@@ -0,0 +1,10 @@
+# News Site
+The project was to create a classic magazine or newspaper site, including a header, a main news article, a grid of news articles. Also to create a responsive design with the following views; Desktop 4 columns, tablet 2 columns, mobile 1 column. 
+
+## The problem
+CSS grid and flexbox was hard to grasp at first. I had a hard time making the items in the header change position in tablet size. I used the study material and help from course mates to solve the coding problems. 
```

Comment:
Not sure exactly what you wanted to achieve with the header, perhaps more similar to the one in the project brief? I also had some problems with the header in the padlet version and was going to go back and work on it later, but I ran out of time. :) I understand that you wanted something different, however I think it looks good.